src/main/resources/db/migration/V1_92__STOKE_STAFFORDSHIRE_RECONFIGURATION_PART_2.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO team (code, name, ldu_code) values('N52SAS', 'IOM Staffordshire North', 'N52STO');
```

Comment:
**--IOM Staff north
UPDATE team set name = 'IOM Staffordshire North', ldu_code = 'N52STO' where code = 'N52SAS';**
You had the above in the first script, did it not exist already?